src/libraries/System.Runtime/tests/System/DelegateTests.cs

```diff
@@ -344,6 +372,32 @@ public static void DynamicInvoke_DefaultParameter_DecimalParameterWithAttributeA
                 (decimal)(new DecimalWithDefaultValueAttribute(DecimalMethod)).DynamicInvoke(new object[] { Type.Missing }));
         }
 
+        // CoreCLR prioritizes CustomConstant over DecimalConstant attribute
+        [Fact]
+        public static void DynamicInvoke_DecimalAndCustomConstantAttribute_DecimalParameterWithAttributeAndMissingValue()
+        {
+            Assert.Equal(
+                new decimal(12, 13, 14, true, 1),
+                (decimal)(new DecimalDelegateWithDecimalAndCustomConstantAttribute(DecimalMethod)).DynamicInvoke(new object[] { Type.Missing }));
+        }
```

Comment:
@jkotas same as for `DynamicInvoke_CustomConstantAndDateTimeAttribute_DateTimeParameterWithMissingValue` the tests started failing on NativeAOT lanes. Please take a look at my comment above.